src/System.Runtime.Intrinsics.X86/src/System/Runtime/Intrinsics/X86/Sse2.cs

```diff
@@ -0,0 +1,545 @@
+// =+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+
+//
+// SSE2.cs
+//
+// A class that implements intrinsic functions to provide access to Intel SSE/SSE2 instructions.
+//
+// =+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+
+
+using System;
+using System.Runtime.Intrinsics;
+
+namespace System.Runtime.Intrinsics.X86
+{
+    /// <summary>
+    /// Provides access to Intel SSE2 hardware instructions via intrinsics
+    /// </summary>
+    /// <para>
+    /// SSE2 class provides access to 128-bit SSE/SSE2 SIMD instructions
+    /// </para>
+
+    public static class Sse2
```

Comment:
RyuJIT requires both. However, this is more of a consistency thing than anything else. There was some discussion on the other thread as well with arguments for both sides (joining or splitting).

Personally, I want a clear distinction between each instruction set and think they should not be combined just because the runtime CoreFX is mostly closely tied to requires it.

I also feel that combining them blurs the lines of the requirements for the design of future additions or architectures.